teamcity-rest-client-impl/src/test/kotlin/org/jetbrains/teamcity/rest/BranchesTest.kt

```diff
@@ -6,73 +6,75 @@ import org.junit.Ignore
-      it.revisions
-      it.getArtifacts()
+    @Test
+    fun test_list_works_no_branches() {
+        kotlinBuildsNoBranches()
+                .withAllBranches()
+                .withStatus(BuildStatus.SUCCESS)
+                .limitResults(20)
+                .all().forEach {
+                    Assert.assertTrue(it.branch.isDefault)
+                }
     }
-  }
 
-  @Ignore("There are no recent changes, because active development has been moved to buildserver")
-  @Test
-  fun test_kotlin_branches() {
-    val branches = mutableSetOf<String>()
-    kotlinBuilds()
-            .withAllBranches()
-            .withStatus(BuildStatus.SUCCESS)
-            .limitResults(50)
-            .all().forEach {
-      branches += it.branch.name!!
-      println(it)
+    @Test
+    fun test_list_works() {
+        kotlinBuilds()
+                .withAllBranches()
+                .withStatus(BuildStatus.SUCCESS)
+                .limitResults(20)
+                .all().forEach {
+                    it.parameters
+                    it.changes.joinToString("\n")
+                    it.revisions
+                    it.getArtifacts()
+                }
     }
 
-    Assert.assertTrue("Actual branches: $branches", branches.size > 1)
-  }
+    @Ignore("There are no recent changes, because active development has been moved to buildserver")
+    @Test
+    fun test_kotlin_branches() {
+        val branches = mutableSetOf<String>()
+        kotlinBuilds()
+                .withAllBranches()
+                .withStatus(BuildStatus.SUCCESS)
+                .limitResults(50)
+                .all().forEach {
+                    branches += it.branch.name!!
+                    println(it)
+                }
 
-  @Test
-  fun test_kotlin_default() {
-    val branches = mutableSetOf<String>()
-    kotlinBuilds()
-            .withStatus(BuildStatus.SUCCESS)
-            .limitResults(50)
-            .all().forEach {
-      branches += it.branch.name!!
-      println(it)
+        Assert.assertTrue("Actual branches: $branches", branches.size > 1)
     }
 
-    Assert.assertTrue("Actual branches: $branches", branches.size == 1)
-  }
+    @Test
+    fun test_kotlin_default() {
+        val branches = mutableSetOf<String>()
+        kotlinBuilds()
+                .withStatus(BuildStatus.SUCCESS)
+                .limitResults(50)
+                .all().forEach {
+                    branches += it.branch.name!!
+                    println(it)
+                }
 
-  private fun kotlinBuilds(): BuildLocator {
-    return publicInstance()
-            .builds()
-            .fromConfiguration(compilerAndPluginConfiguration)
-  }
+        Assert.assertTrue("Actual branches: $branches", branches.size == 1)
+    }
 
-  private fun kotlinBuildsNoBranches(): BuildLocator {
-    return publicInstance().builds()
-            .fromConfiguration(compileExamplesConfiguration)
-  }
+    private fun kotlinBuilds(): BuildLocator {
```

Comment:
Why these functions are still named "kotlinBuilds"? They don't refer to Kotlin build configurations anymore as far as I can see.

Reply:
Fixed.